checkbox-core-snap/prepare.sh

```diff
@@ -47,3 +47,5 @@ echo "Copying over checkbox-ng to $series"
 rsync -r --links ../checkbox-ng $series/
 echo "Copying over checkbox-support to $series"
 rsync -r --links ../checkbox-support $series
+echo "Dumping version in version file for $series"
+(cd .. && python3 -m setuptools_scm | grep -oP "\S+$") 2>/dev/null 1>$series/version.txt
```

Comment:
Out of curiosity, what's the grepping achieving?

Reply:
on old versions of setuptools_scm instead of just printing the calculated version the output is something like `Caluclated version vX.XX...`, with grep just takes the last part of that